Change default authentication system from RbAcl to WebAC

whikloj opened the pull request:
Addresses [FCREPO-1735](https://jira.duraspace.org/browse/FCREPO-1735)

whikloj committed:
Change default authentication system from RbAcl to WebAC

whikloj committed:
Include default username/password for triplestore and Solr indexing.

awoods commented on install_scripts/fedora_camel_toolbox.sh:
```diff
@@ -15,3 +15,8 @@ cd $HOME_DIR
 
 sed -i 's|solr.baseUrl=localhost:8983/solr/collection1|solr.baseUrl=localhost:8080/solr/collection1|' /opt/karaf/etc/org.fcrepo.camel.indexing.solr.cfg
+if [ "$FEDORA_AUTH" == "true" ]; then
+  sed -i 's|fcrepo.authUsername=$|fcrepo.authUsername=fedoraAdmin|' /opt/karaf/etc/org.fcrepo.camel.indexing.solr.cfg
```
This all works as expected, but the admin username and password `sed` commands are also needed for the following:
- org.fcrepo.camel.reindexing.cfg
- org.fcrepo.camel.indexing.triplestore.cfg
- org.fcrepo.camel.fixity.cfg

whikloj committed:
Use karaf client -f to read file of commands
Add checks for files and attempt to re-install features if not installed.

whikloj committed:
Remove fcrepo-audit feature from karat script.
Change order of commands passed to karaf client
Add -u -h & -a arguments to karat client

awoods commented:
Resolved with:
https://github.com/fcrepo4-exts/fcrepo4-vagrant/commit/0c6e9a13311bc9c4d7e881367b5bfef0a619e08e and
https://github.com/fcrepo4-exts/fcrepo4-vagrant/commit/982af09fe88d214ffdfc810dc062c43bf124262b
